src/cmps/tags/Tags.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from "react";
+import './Tags.css';
+import Tag from '../tag/Tag';
+// eslint-disable-next-line
+import { string } from "prop-types";
+
+
+class Tags extends Component {
+
+    state = {
+        tags: [{
+            id: 1,
+            name: 'tag1'
+        }, {
+            id: 2,
+            name: 'tag2'
+        }]
+    }
+
+    remove = () => {
+        console.log('button X clicked');
+    }
+
+    addTag = (event: any) => {
```

Comment:
avoid using `any`, there is no point for using typescript this way 🤓

Reply:
Do not know how to define event...

Reply:
me neither but Google knows:
`React.FormEvent<HTMLInputElement>`

Reply:
https://stackoverflow.com/questions/42081549/typescript-react-event-types